src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -333,3 +333,56 @@
     type: group
     short-summary: Manage resource tags.
 """
+helps['resource link'] = """
+    type: group
+    short-summary: Manage links between resources.
```

Comment:
A long-summary here would be a good place to elaborate on what a link between resources means, or at least point to some documentation that does the same.